importScripts/sampleRecords/library.destination.json

```diff
@@ -15,7 +14,7 @@
       "ca_objects.LastEditBy":"LanK",
       "ca_objects.LastEditDate":"22 March 2004",
       "ca_objects.LibraryNumber":"AAM",
-      "ca_objects.PublicationType":"Books",
+      "ca_objects.PublicationType":"Book",
```

Comment:
This was a client request, and this is the result of https://github.com/rwahs/installation-profile/pull/54